### ISSUE TITLE
Resurrection doesn't take the "path" parameter into account

### ISSUE THREAD

**Issue by dav3860** [logstash-plugins/logstash-output-elasticsearch#529]
It seems that the resurrection feature always tries to query elasticsearch with a "HEAD /", event though a path, like "/es", is explicitly defined in the output configuration. So it doesn't work.

- Version: 5.1.1
- Operating System: Centos 7
- Config File (if you have sensitive info, please remove it):
```
 elasticsearch {
    hosts => ["myserver.domain.local:443"]
    path => "/es"
    index => "logstash-%{type}-%{+YYYY.MM.dd}"
    ssl => true
    ssl_certificate_verification => false
    template_overwrite => true
    template => "/etc/logstash/templates/elasticsearch-template.json"
    flush_size => 1000
    document_type => "%{type}"
}
```

- Sample Data:
These warning appear in the logs :
`[2016-12-15T14:54:44,388][WARN ][logstash.outputs.elasticsearch] Attempted to resurrect connection to dead ES instance, but got an error. {:url=>#<URI::HTTPS:0x1bffd71d URL:https://myserver.domain.local/es/>, :error_type=>LogStash::Outputs::ElasticSearch::HttpClient::Pool::BadResponseCodeError, :error=>"Got response code '403' contact Elasticsearch at URL 'https://myserver.domain.local/'"}`

Related reverse proxy logs:
```
172.16.x.y - - [15/Dec/2016:14:51:18 +0100] "POST /es/_bulk HTTP/1.1" 200 879
172.16.x.y - - [15/Dec/2016:14:51:18 +0100] "HEAD / HTTP/1.1" 403 -
172.16.x.y - - [15/Dec/2016:14:51:18 +0100] "POST /es/_bulk HTTP/1.1" 200 506
```

- Steps to Reproduce:
Put elasticsearch behind a reverse proxy, but not behind /.

**Comment by andrewvc** [comment]
Interesting. Thanks for finding this. Definitely a bug, I'll dig into this today.

**Comment by andrewvc** [comment]
@dav3860 this will be fixed when #450 is merged and released. Thanks very much for the report.

**Comment by andrewvc** [comment]
Closing now that #450 is released.
